SPECS/erlang/erlang.spec

```diff
@@ -18,12 +18,12 @@ BuildRequires:  unzip
 erlang programming language
```

Comment:
Just a nit. 
can we add this here? "Erlang is a programming language and runtime system for building massively scalable soft real-time systems with requirements on high availability."

Reply:
Updated